### ISSUE TITLE
Generalise styles across plugins

### ISSUE THREAD

**Issue by bedeoverend** [SimplaElements/simpla-richtext-behavior#27]
Currently we've got a few different stylesheets being injected by plugins e.g. placeholder styles, pre-wrap styles. Plus there're also styles that should be applied to images, which is being done over on [`simpla-article`](https://github.com/SimplaElements/simpla-article/blob/master/src/simpla-article.html#L25-L35) but in reality should be done in this behavior.

All the plugins should be able to define styles that are (conditionally) applied, and all be injected into the one stylesheet. That stylesheet should also only be applied [once per root](https://github.com/SimplaElements/simpla-text/issues/40) to avoid unnecessary stylesheets.

**Comment by madeleineostoja** [comment]
> That stylesheet should also only be applied once per root to avoid unnecessary stylesheets

FWIW that should be a future enhancement, shouldn't hold up implementation of this

**Comment by bedeoverend** [comment]
Yup definitely 👍 can split it into a separate issue once core problem is resolved, if needed
